impl/src/test/java/org/jboss/forge/test/roaster/model/MethodModifierTest.java

```diff
@@ -59,8 +59,8 @@ public void testDefaultModifier() throws Exception
       Assert.assertThat(method.toString(), containsString("public default void test()"));
       Assert.assertThat(method.getBody(), equalTo(""));
       String body = "int a=2;";
-      method.setDefault(false).setBody(body);
-      Assert.assertFalse(method.isDefault());
```

Comment:
setDefault(false).setBody() should throw an exception, I prefer to change false to true to keep test